src/main/java/tr/edu/ogu/ceng/controller/CompanyController.java

```diff
@@ -0,0 +1,36 @@
+package tr.edu.ogu.ceng.controller;
+
+
+import java.util.List;
+import org.springframework.http.MediaType;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import tr.edu.ogu.ceng.model.Company;
+import tr.edu.ogu.ceng.service.CompanyService;
+
+@RestController
+@RequestMapping(value="/api/company",produces = MediaType.APPLICATION_JSON_VALUE)
+public class CompanyController {
+	@Autowired
+    CompanyService companyService;
+     
+    @PutMapping("/{id}")
+	public ResponseEntity<Company> updateCompany(@RequestBody Company company, @PathVariable(name="id") int id) {
```

Comment:
Neden id obje ile birlikte gelmedi de ayrı bir paramtre olarak geliyor.

Reply:
o kısım benim değil hocam efekanın kısmı sanırım

Reply:
evet ama senin commitinde değişiklik olarak görünüyor, Efekan'ın değişiklikleri kabul edilince burada görünmeyecektir. Ancak efekan değişiklik yaparsa senin için başka bir kaos oluşacaktır. O yüzden main branchinden yeni branch oluşturmak daha mantıklı olurdu.

Reply:
tamam hocam, peki update kısmını tamamen silersem kendi kodumdan efekanın koduna bir etkisi olur mu merge ettikten sonra

Reply:
evet Efekan eklemiş sen de silmiş olursun :)

Reply:
senin birşey yapmana gerek yok Efekan düzeltecektir onları. Sen kendin düzeltip conflict olan dosyaları düzeltmen yeterli.

Reply:
@yagizharman hayır artık bunu yapmana gerek yok. bir dahaki sefere buna dikkat edersin sadece önce Efekan'ın PR'ının geçmesi gerekecek tabi sen de düzeltip commitleyebilirsin.